src/storage/index.js

```diff
@@ -13,22 +13,26 @@ const s3 = new aws.S3({
   credentials: config.s3.credentials,
 });
 
-const getFileUpload = (fileTag, maxFileSize) => {
-  const fileUpload = multer({
-    storage: multerS3({
-      s3,
-      bucket: config.s3.bucket,
-      acl: 'public-read',
-      key: (req, file, next) => {
-        next(null, `portal/profiles/${req.user.uuid}${path.extname(file.originalname)}`);
-      },
-    }),
+const uploadToS3 = async (mediaType, file, uuid) => {
+  const mediaTypeConfig = Media.getMediaTypeConfig(mediaType);
+  const params = {
+    ACL: 'public-read',
+    Body: file.buffer,
+    Bucket: config.s3.bucket,
+    Key: `portal/${mediaTypeConfig.uploadPath}/${uuid}${path.extname(file.originalname)}`,
```

Comment:
Nit: include `portal/` as part of the configured upload path. Maybe set like a `BASE_UPLOAD_PATH` or something since it's a single bucket for all ACM stuff and then in the config file, prepend the base path to the other upload paths.